remote_attestation_sessions_client/src/lib.rs

```diff
@@ -21,8 +21,8 @@ use oak_remote_attestation::handshaker::{
 };
 use oak_remote_attestation_sessions::SessionId;
 
-#[async_trait]
 /// Abstract version of networking stub.
+#[async_trait(? Send)]
```

Comment:
Async trait requires the definition and all implementations to be marked as optionally sync if one implementation is not.